src/main/java/com/microsoft/azure/relay/RelayConnectionStringBuilder.java

```diff
@@ -244,13 +244,10 @@ private void parseConnectionString(String connectionString) {
 			} else if (key.equalsIgnoreCase(SHARED_ACCESS_SIGNATURE_CONFIG_NAME)) {
 				this.sharedAccessSignature = value;
 			} else if (key.equalsIgnoreCase(OPERATION_TIMEOUT_CONFIG_NAME)) {
-				// TODO: handle duration strings in the C# TimeSpan format
 				try {
-					Duration timeValue = Duration.parse(value);
-					this.operationTimeout = timeValue;
-				} catch (DateTimeParseException e) {
-					throw new DateTimeParseException(value + " cannot be parsed into a valid duration.",
-							e.getParsedString(), e.getErrorIndex());
+					this.operationTimeout = Duration.ofSeconds(Long.parseLong(value));
+				} catch (NumberFormatException e) {
+					throw RelayLogger.throwingException(e, this);
 				}
 			} else {
 				throw new IllegalArgumentException("the following is not a valid field for connection string: " + key);
```

Comment:
>throw [](start = 4, length = 5)

[Unrelated to these changes] While I'm here do we want to have RelayLogger.throwingException wrapper around this?